### PR TITLE
Start implementation of quota exceeded check

### PR DESCRIPTION
**Pull request by JanAckermann**
## Description
Bugfix: Show message while upload size exceeds quota

We fixed a bug where an upload silently failed if the upload size exceeds the space quota. It now displays a detailed message instead
will be shown.

image

## Related Issue

- Fixes https://github.com/owncloud/web/issues/7025

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: changelog/unreleased/bugfix-show-message-while-upload-size-exceeds-quota [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Bugfix: Show message while upload size exceeds quota
+
+We fixed a bug where an upload silently failed, while the upload size exceeds the space quota, now a detailed message
```
Comment (pascalwengerter) [inline changelog/unreleased/bugfix-show-message-while-upload-size-exceeds-quota:3]:
```suggestion
We fixed a bug where an upload silently failed if the upload size exceeds the space quota. It now displays a detailed message instead
```

File: changelog/unreleased/bugfix-show-message-while-upload-size-exceeds-quota [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Bugfix: Show message while upload size exceeds quota
+
+We fixed a bug where an upload silently failed if the upload size exceeds the space quota. It now displays a detailed message instead
+will be shown.
```
Comment (JammingBen) [inline changelog/unreleased/bugfix-show-message-while-upload-size-exceeds-quota:4]:
```suggestion

```

**Review by JammingBen — APPROVED** [review]
LGTM, minor thing in the changelog item.

**Comment by pascalwengerter** [comment]
@JanAckermann could you provide some feedback on https://github.com/owncloud/ocis/issues/3767#issuecomment-1135705272 whether the reva/ocis change is relevant for web still? Perhaps it isn't after this PR is merged but still valuable for other clients :)

**Comment by JanAckermann** [comment]
@pascalwengerter I don't think it is relevant for web, as we are checking the quota on the personal drive as well, which will AFAIK shrink as the available disk size will

**Pull request merged**